Server/main.py

```diff
@@ -31,13 +34,34 @@ def login():
         return response
     return 'error', 500
 
+#Routes
+@app.route('/api/createUser', methods=['POST'])
+def create_user():
+    status = 401
+    if not request.is_json:
+        return "Invalid Content Type", 500
+    if not auth_verification(request):
+        return "Not Authorized", 401
+
+    json_data = request.get_json()
+    confirm_password = json_data['confirmPassword']
+    password = json_data['password']
+    if confirm_password != password:
+        return 'Passwords do not match', 200
+    new_user = User(username=json_data['username'], password=json_data['password'], firstName=json_data['firstname'],
+                    lastName=json_data['lastname'], employeeId=json_data['employeeId'], isAdmin=json_data['isAdmin'])
+    did_Add = app_users.add_user(new_user)
+    if not did_Add:
+        return 'Username already exists', 200
+    return 'No Content', 201
+
+
+#Tes Routes
 @app.route('/api/test', methods=['GET'])
 def test():
     print('Started')
-    if 'auth' not in request.headers:
-        return 'Invalid Session', 500
     token = request.headers['auth']
```

Comment:
In the test endpoint:
remove the token reference and ensure only auth_verification gets the request. like so: auth_verification(request)